src/app/shared/keycloak.service.ts

```diff
@@ -74,7 +74,7 @@ export class KeycloakService {
     if (this.skip) {
       return true;
     }
-    return this.auth.authz.tokenParsed;
+    return this.auth.authz && this.auth.authz.tokenParsed;
```

Comment:
is this really the best solution? it's set in the init, better would be to find out why / how it was null